R/Reactant/build_tarballs.jl

```diff
@@ -208,6 +208,11 @@ if [[ "${bb_full_target}" == *mingw* ]]; then
     mv ${libdir}/libReactantExtra.so ${libdir}/libReactantExtra.dll
 fi
 cp -v bazel-bin/*.jl ${prefix}
+
+# process debug symbols
+if [[ "${bb_full_target}" == *mode\+dbg* ]]; then
```

Comment:
What happened to this suggestion? It appears it was reverted. Also, there are merge conflicts which needs to be resolved

Reply:
Nono, I applied it and later I moved it inside another `if`. Should be ok now.

Reply:
Is it? https://github.com/JuliaPackaging/Yggdrasil/blob/0523ca1aed988a3507084a6e21d35587b45de612/R/Reactant/build_tarballs.jl#L208

Reply:
Yep, because the outer `if` is for Apple platforms only:
https://github.com/JuliaPackaging/Yggdrasil/blob/0523ca1aed988a3507084a6e21d35587b45de612/R/Reactant/build_tarballs.jl#L204-L208